setup.py

```diff
@@ -42,7 +42,8 @@ def find_scripts(script_dir, prefix):
     install_requires=[
         'astropy',
         'ctapipe~=0.7.0',
-        'eventio~=1.0',
+        'ctaplot',
+        'eventio~=0.20.3',
```

Comment:
why changing the version of `eventio`?

Reply:
This is the version required by ctapipe 0.7. It was actually broken before (running scripts not possible as it would either complain about it being too old for lstchain or too new for ctapipe)

Reply:
Don't know why it worked on travis.